setup.py

```diff
@@ -55,6 +55,6 @@
     test_suite='tests',
     tests_require=test_requirements,
     url='https://github.com/kylebarron/keplergl_cli',
-    version='0.3.3',
+    version='0.3.4',
```

Comment:
You shouldn't change the version here; we'll have a separate PR to bump the version, which will probably be 0.4.0